crates/types/src/config/common.rs

```diff
@@ -193,6 +207,15 @@ impl CommonOptions {
                 .join(DEFAULT_STORAGE_DIRECTORY)
         })
     }
+
+    pub fn rocksdb_bg_threads(&self) -> NonZeroU32 {
+        self.rocksdb_bg_threads.unwrap_or(
+            std::thread::available_parallelism()
+                .unwrap_or(NonZeroUsize::new(3).unwrap())
```

Comment:
Out of curiosity why 3 and not 2 like for the high priority bg threads?

Reply:
Memtable flushes happen in high-priority threads, those are IO bound and parallelism might hurt IO, compactions are CPU bound and they happen on the low priority threads, so concurrency helps.

All in all, those are arbitrary, but I tried to somewhat follow the defaults of rocksdb.